PubHub/PubHub.Common/Services/UserService.cs

```diff
@@ -69,14 +70,15 @@ public async Task<ServiceResult<UserInfoModel>> GetUserAsync(Guid userId)
         /// <returns>A list of <see cref="BookInfoModel"/></returns>
         /// <exception cref="ArgumentException"></exception>
         /// <exception cref="NullReferenceException"></exception>
-        public async Task<ServiceResult<IReadOnlyList<BookInfoModel>>> GetUserBooksAsync(Guid userId)
+        public async Task<ServiceResult<IReadOnlyList<BookInfoModel>>> GetUserBooksAsync(Guid userId, BookQuery queryOptions = null!)
```

Comment:
We don't have to change it, but wouldn't you normally use `BookQuery?` and assign `null` instead of suppressing nullable warnings with `!`?

Reply:
You would.
It' actually better to use `BookQuery?` instead of suppressing `null` as it signifies intent 😊

I'll change it for good measure 👌

Reply:
Should be done now.